dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -69,7 +69,7 @@ class DALIGenericIterator(object):
                  for data or label correspondingly.
                  output_names should be distinct.
     size : int
-           Epoch size.
+          Number of samples in the epoch in the whole data set.
```

Comment:
> in the whole data set.
Why? This is what user usually wants to do but you can set it to anything else (e.g. for testing).

Reply:
Done